export/distro/format.go

```diff
@@ -38,3 +38,34 @@ func (xmlTr xmlFormater) Format(event *models.Event) []byte {
 	}
 	return b
 }
+
+type thingsboardJSONFormater struct {
```

Comment:
Minor typo -- should have two "tt"s. thingsboardJSONFormatter

Reply:
Completely agree. But, should I fix all other "formater"?